commitizen/cz/conventional_commits/conventional_commits.py

```diff
@@ -213,3 +215,104 @@ def process_commit(self, commit: str) -> str:
         if m is None:
             return ""
         return m.group(3).strip()
+
+
+class ConventionalCommitsTextile(ConventionalCommitsCz):
```

Comment:
Would it be possible to make this 'changelog providers' not tied to conventional commits? How would it work for any other plugin?